mysql-plugin/src/e2e-test/features/mysqlsink/DesignTime.feature

```diff
@@ -0,0 +1,80 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License..
+
+Feature: MySQL Sink - Design time scenarios
+
+  @TS-MYSQL-SINK-DSGN-01 @MYSQL_SOURCE_TEST
```

Comment:
Why do we require the tags "TS-MYSQL-SINK-DSGN-**" ?

Reply:
There is no such specific use ...Should we remove it?

Reply:
Yes.

Reply:
Removed the tags.